integration-tests/ccip-tests/actions/ccip_helpers.go

```diff
@@ -4036,7 +4035,7 @@ func (c *CCIPTestEnv) ConnectToDeployedNodes() error {
 		for _, chainlinkNode := range c.LocalCluster.ClCluster.Nodes {
 			c.nodeMutexes = append(c.nodeMutexes, &sync.Mutex{})
 			c.CLNodes = append(c.CLNodes, &client.ChainlinkK8sClient{
-				ChainlinkClient: chainlinkNode.API.WithRetryCount(3),
```

Comment:
What's the reason for removing the retry?

Reply:
This seems to be an ancient kludge from when our deployments were much more unstable. We haven't had this function call for a long time in the core repo.